pulseapi/users/test_views.py

```diff
@@ -46,4 +46,6 @@ def test_user_status_true(self):
         json_obj = json.loads(string)
         self.assertEqual(json_obj['loggedin'], True)
         self.assertEqual(json_obj['email'], 'test@example.org')
+        self.assertTrue(json_obj['profileid'])
+        self.assertIsInstance(json_obj['profileid'], type("a"))
```

Comment:
I get what this line is doing but it feels odd. Is this some convention I'm not familiar with, or is there a more direct way to check if something is a string in Python? @gideonthomas

Reply:
I think i have found a better way to do this.